### PR TITLE
Chore/TransactionsEndpoints/Loc 

### PR DESCRIPTION
**Pull request by locbgiang**
First commit for transaction endpoints.
create new transaction
view with transaction id
view list with profileId
update transaction
delete transaction

### CODE REVIEW

**Review by imriven — APPROVED** [review]
This looks good to me all of the routes look useful!

File: api/transaction/transactionRouter.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+const express = require('express');
+const router = express.Router();
+const Transaction = require('./transactionModel');
+
+// create transaction
+router.post('/', (req, res) => {
+  const transactionData = req.body;
+  console.log('aaaaaaaaaaaaaaaa');
```
Comment (chelsabeth) [inline api/transaction/transactionRouter.js:8]: let's get rid of this console.log before we merge :)

**Review by chelsabeth — CHANGES_REQUESTED** [review]
Looks good besides the console.log that I commented on 👍

**Review by imriven — APPROVED** [review]
Looks good!

**Review by chelsabeth — APPROVED** [review]
Looks clean and concise 👍 Good work!

**Review by chelsabeth — APPROVED** [review]
Looks good 👍

**Review by imriven — APPROVED** [review]
Yup looking good!

**Pull request merged**